todoman/cli.py

```diff
@@ -171,10 +171,12 @@ def cli(ctx, color, porcelain):
 @click.argument('summary', nargs=-1)
 @click.option('--list', '-l', callback=_validate_list_param,
               help='The list to create the task in.')
+@click.option('--category',
+              help='Categories of todo')
```

Comment:
Please have a quick look at `_todo_property_options`. We basically have `option`s which are common to `new` and `edit` there, so they apply to both places. This `option` should be moved there to keep the code simpler.